aiohttp/web_fileresponse.py

```diff
@@ -198,13 +198,26 @@ def __init__(self, path: Union[str, pathlib.Path],
     ) -> Optional[AbstractStreamWriter]:
         filepath = self._path
 
+        encoding = None
+        accept_encoding = request.headers.get(hdrs.ACCEPT_ENCODING, '')
+
+        brotli = False
+        if 'br' in accept_encoding:
```

Comment:
You also need some check for whether brotli dist is installed in the env at all.

Reply:
Not really. We are just serving the raw compressed bytes as they are found on disk. The client requesting them (with indication of support for brotli in the headers) does the decoding. This client might not even be a Python script, like Chrome.